getting-started.md

```diff
@@ -3,3 +3,4 @@
 - This is the entrypoint for handling your validator keystores
 - Web3signer takes care for signing requests of your validator
 - Web3signer supports client diversity, working with different clients such as [Prysm](http://my.dappnode/#/installer/prysm.dnp.dappnode.eth), [Lighthouse](http://my.dappnode/#/installer/lighthouse.dnp.dappnode.eth), [Teku](http://my.dappnode/#/installer/teku.dnp.dappnode.eth) and [Nimbus](http://my.dappnode/#/installer/nimbus.dnp.dappnode.eth)
+- The Flyway container takes care of importing the previous version's slashing protection database when an update is installed, so it's meant to run only once.
```

Comment:
```suggestion
- The Flyway service takes care of importing the previous version's slashing protection database when an update is installed, so it's meant to run only once per update, and it is normal to see that it the Flyway service is stopped
```
we dont describe the other services in this section so it felt out of place but yeah good catch this makes sense to keep from making people think they have a problem when the Flyway service is listed as stopped, because its supposed to

Reply:
the getting started's for all signers need to be updated as this whole thing is outdated (except this update about the flyway) also we shouldnt have links or anything to the installers as thats not how we install them anymore. so once this is merged ill work on a new Uniform message for all the signers with some more modern info like the stakersUI etc